src/draw/drawings.js

```diff
@@ -71,18 +74,18 @@ function createNonTraversableCellSprites(
  * the corresponding cells in green on the map.
  */
 export function drawPlannedPath(path, cpt, hexColor = 0x00ff00, alpha = 0.25) {
-  clearSprites(tagpro.renderer.pathSprites); // clear the previous path from the map
+  pathSprites = clearSprites(pathSprites); // clear the previous path from the map
```

Comment:
no, because they have to exist the next time drawPlannedPath is called. Each time drawPlannedPath is called, it needs to clear the previously drawn sprites before drawing new ones, so the reference has to persist outside the function scope. With that info, do you have a recommended modification to this code?

Note: I realize clearSprites will always return `[]` or `undefined`. Personally, I like the idea of clearSprites telling the user what the variable you passed to it should be after calling it though.